test/views/declaration/LinkDucrToMucrViewSpec.scala

```diff
@@ -117,5 +118,14 @@ class LinkDucrToMucrViewSpec extends UnitViewSpec with CommonMessages with Injec
       }
     }
 
+    onEveryDeclarationJourney(withStatus(DeclarationStatus.AMENDMENT_DRAFT)) { implicit request =>
+      val view = createView()
+
+      "hide 'Back' button" in {
```

Comment:
Need to specify that this is testing the situation where the draft dec is an amendment draft